sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubClient/EventHubClientLiveTests.cs

```diff
@@ -253,9 +254,9 @@ await using (var client = new EventHubClient(connectionString))
 
                     await Task.Delay(TimeSpan.FromSeconds(5));
 
-                    Assert.That(async () => await client.GetPartitionIdsAsync(), Throws.TypeOf<ObjectDisposedException>());
-                    Assert.That(async () => await client.GetPropertiesAsync(), Throws.TypeOf<ObjectDisposedException>());
-                    Assert.That(async () => await client.GetPartitionPropertiesAsync(partition), Throws.TypeOf<ObjectDisposedException>());
+                    Assert.That(async () => await client.GetPartitionIdsAsync(), Throws.TypeOf<EventHubsObjectClosedException>().Or.TypeOf<ObjectDisposedException>());
```

Comment:
Differences in underlying AMQP foundation; extending to allow both track one and track two stacks to be used.

Reply:
Maybe we should add a note somewhere to remind us of keeping only the `EventHubsObjectClosedException` when we completely remove Track One code. I'm totally ok if you're already keeping track of it.

Reply:
Good call.

I've got a few issues that I need to open;  I've got this tracked as part of the scope to remove the track one code and will be sure to detail it that issue when I open it.